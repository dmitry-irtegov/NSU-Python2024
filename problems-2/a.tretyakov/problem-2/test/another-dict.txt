one - uno, dos, tres, quatro
two - uno, dos, tres, quatro
three - uno, dos, tres, quatro
four - uno, dos, tres, quatro
five - uno, dos, tres, quatro
six - uno, dos, tres, quatro
seven - uno, dos, tres, quatro
eight - uno, dos, tres, quatro
nine - uno, dos, tres, quatro
ten - uno, dos, tres, quatro