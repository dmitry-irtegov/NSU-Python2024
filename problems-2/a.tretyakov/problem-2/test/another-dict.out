dos - one, two, three, four, five, six, seven, eight, nine, ten
quatro - one, two, three, four, five, six, seven, eight, nine, ten
tres - one, two, three, four, five, six, seven, eight, nine, ten
uno - one, two, three, four, five, six, seven, eight, nine, ten
